src/reflect/scala/reflect/internal/util/Statistics.scala

```diff
@@ -267,19 +308,20 @@ quant)
    */
   final val hotEnabled = false
```

Comment:
To get a better look at the performance overhead (or absense thereof), perhaps you could extend the switchpoint technique to `hotEnabled`.